pythoncalculator/divide.py

```diff
@@ -0,0 +1,2 @@
+def divide(x, z):
```

Comment:
I think z should be y here on line 1
```suggestion
def divide(x, y):
```